cmd/kubeadm/app/util/config/cluster.go

```diff
@@ -125,7 +125,7 @@ func getInitConfigurationFromCluster(kubeconfigDir string, client clientset.Inte
 		// However, if newControlPlane == true, initcfg.NodeRegistration is not used at all and it's overwritten later on.
 		// Thus it's necessary to supply some default value, that will avoid the call to DetectCRISocket() and as
 		// initcfg.NodeRegistration is discarded, setting whatever value here is harmless.
-		initcfg.NodeRegistration.CRISocket = constants.DefaultDockerCRISocket
+		initcfg.NodeRegistration.CRISocket = "/var/run/unknown.sock"
```

Comment:
@SataQiu based on the comments above the assignment this seems fine.

looks like @rosti is on vacation this week, so i'm going to LGTM / approve this, but let's iterate again in the 1.20 cycle if more changes are needed.

thanks.